Project.toml

```
@@ -1,6 +1,6 @@
 name = "KernelFunctions"
 uuid = "ec8451be-7e33-11e9-00cf-bbf324bd1392"
-version = "0.10.5"
+version = "0.11.5"
```

Probably this should be
```suggestion
version = "0.10.6"
```
since it is a non-breaking new feature :slightly_smiling_face: